requirements.txt

```diff
@@ -3,8 +3,9 @@ prometheus-client
 requests
 six
 PyYAML
-Flask==1.1.1
+Flask==1.1.2
 gevent
 cffi
 beautifulsoup4==4.9.3
-lxml
\ No newline at end of file
+lxml
+itsdangerous==1.1.0
```

Comment:
This is not being used, is it?

Reply:
Look at the failing tests above. This version looks like a needed dependency.

Reply:
At least for the PyTest.

Reply:
Ok, implicit dependency through Flask, I see.